core/domain/exp_services_test.py

```diff
@@ -1673,6 +1682,17 @@ class ExplorationCommitLogUnitTests(ExplorationServicesUnitTests):
         'post_commit_status': 'public'
     }
 
+    COMMIT_UPDATE_FIRST_PUBLISHED_MSEC_EXP_2 = {
```

Comment:
I just realized that first_published has to be outside a publish event for the one off job. We have to have separate commits in order to update the first_published_msec outside the actual publish event. What are your thoughts?